src/gui/org/deidentifier/arx/gui/view/impl/utility/ViewClassificationAttributes.java

```diff
@@ -233,9 +231,11 @@ private void update() {
             item.dispose();
         }
         
-        for (int i = 0; i < handle.getNumColumns(); i++) {
-            createTableItem(handle, i, features, model.getSelectedFeatures());
-            createTableItem(handle, i, classes, model.getSelectedClasses());
+        for (int col = 0; col < handle.getNumColumns(); col++) {
+            String attribute = handle.getAttributeName(col);
+            Image image = controller.getResources().getImage(model.getInputDefinition().getAttributeType(attribute));
+            createTableItem(attribute, image, features, model.getSelectedFeatures());
+            createTableItem(attribute, image, classes, model.getSelectedClasses());

```

Comment:
I don't think that it is necessary to show the image in both tables. I suggest to display it only in the left table on the left side.